thirdparty/lpeg/CMakeLists.txt

```diff
@@ -9,9 +9,10 @@ enable_language(C)
 assert_var_defined(CC)
 assert_var_defined(DYNLIB_CFLAGS)
 assert_var_defined(LUA_DIR)
-assert_var_defined(LUAJIT_LIB)
 
-set(BUILD_CMD_STR "${CC} ${DYNLIB_CFLAGS} -I${LUA_DIR}/src ${LUAJIT_LIB} -o lpeg.so lpcap.c lpcode.c lpprint.c lptree.c lpvm.c")
+# assuming lpeg.so is installed under $(OUTPUT_DIR)/rocks/lib/lua/5.1 and
+# libluajit.so is in $(OUTPUT_DIR)/libs
+set(BUILD_CMD_STR "${CC} ${DYNLIB_CFLAGS} -I${LUA_DIR}/src -Wl,-rpath,'../../../../libs' -lluajit -o lpeg.so lpcap.c lpcode.c lpprint.c lptree.c lpvm.c")
```

Comment:
I see. According to http://stackoverflow.com/questions/24573732/difference-between-relative-path-and-using-origin-as-rpath, current working path will be used as the base dir for relative path like `libs`, which in our case will be root of koreader directory.

So I think for us, the proper setup should be using `$ORIGIN` for all libraries in **koreader/libs** dir and `libs` for libraries that have deeper location like **koreader/common**. Thoughts?